index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
+      integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+    <link rel="stylesheet" href="style.css">
+    <title>DOGS</title>
+  </head>
+  <main>
+<!-- nav bar -->
+  <nav class="navbar">
+    <ul class="ul">
+     <li class="li"><a class="custom" class="active" href="#top">Home</a></li>
+     <li class="li"><a class="custom" href="pages/Tess.html">Tess</a></li>
+     <li class="li"><a class="custom" href="pages/Ruby.html">Ruby</a></li>
+     <li class="li"><a class="custom" href="pages/Photos.html">Photos</a></li>
+     <!-- <li class="li"><a class="custom" href="pages/Videos.html">Videos</a></li> -->
+    </ul>
+  </nav>
+
+  <body>
+
+    <h1 class="title">My Dogs</h1>
+    <div class="container">
+      <div class="row">
+        <div class="col-md-3 col-md-offset-1"><a href="pages/Tess.html"><img src="images/Tess2.jpg" alt="Dog"></a></div>
+        <div class="col-md-3"><a href="pages/Ruby.html"><img src="images/Ruby1.jpg" alt="Dog"></a></div>
+        <div class="col-md-3"><a href="pages/Photos.html"><img src="images/both3.jpg" alt="Dog"></a></div>
+      </div>
+      <div class="row">
+        <div class="col-md-3 col-md-offset-1"><h2 class="title">TESS</h2></div>
```

Comment:
Don't leave something like this, which looks like a navbar, in unless it is functional. Make these links go to the appropriate pages or take it out.